scripts/travis_script

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Travis build script, cf.
+# https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps.
+
```

Comment:
I think this script should also use "set -ev" from the link.

Reply:
I've added this to this change for now, but will likely have to revert again because -v seems fairly useless (-x would seem better), and, more importantly, -e causes the script to fail (jdk_switcher is a function and somewhere in there it encounters an exit code 1).

Reply:
Oh, interesting, "jdk_switcher use ..." calls "sudo update-java-alternatives ...", but "sudo" would only work if .travis.yml contains "sudo: true".

Reply:
Now using -o xtrace (instead of -o verbose) and JAVA_HOME="$(jdk_switcher home openjdk8)" ... (instead of jdk_switcher use ...). This looks better. PTAL.